roles/ovirt-host-deploy-libvirt-guests/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Fetch info about VDSM
+  command: bash -c "rpm -qi vdsm | grep -oE 'Version\\s+:\\s+[0-9\\.]+' | awk '{print $3}'"
+  changed_when: false
+  register: vdsm_version
+
+- block:
+    - name: Change default timeout if setup
+      lineinfile:
+        path: /etc/sysconfig/libvirt-guests
+        regexp: "^SHUTDOWN_TIMEOUT="
+        line: "SHUTDOWN_TIMEOUT={{ host_deploy_shutdown_timeout }}"
+      when:
+        - host_deploy_shutdown_timeout is defined
+
+    - name: Change run in parallel if setup
+      lineinfile:
+        path: /etc/sysconfig/libvirt-guests
+        regexp: "^#PARALLEL_SHUTDOWN="
+        line: "PARALLEL_SHUTDOWN={{ host_deploy_shutdown_parallel }}"
+      when:
+        - host_deploy_shutdown_parallel is defined
+
+    - name: Stop libvirt-guests service if running
```

Comment:
stop? Why?

Reply:
Name says stop, definition says start...

Reply:
Hmm, shouldn't the service be rather restarted? If it's started already before this role invocation and we will change the options, the service won't be restarted using this code ...